contracts/core/extensions/CoreInternal.sol

```diff
@@ -240,7 +246,26 @@ contract CoreInternal is
     )
         external
         onlyOwner
+        timeLockUpgrade
     {
         state.signatureValidator = _signatureValidator;
     }
+
```

Comment:
Why not put this in a constructor instead of having a function that only gets called once?

Reply:
It kills 2 birds with 1 stone, which is 1) adding an initial period where the timelock doesn't apply (not setting it has that effect) and 2) allows adding that timelock

Reply:
I see. That makes sense